tests/common/helpers/pfc_gen.py

```diff
@@ -122,7 +122,7 @@ def main():
     pause time      |        0x0000         |
                     -------------------------
     """
-    src_addr = "\x01\x02\x03\x04\x05\x06" 
+    src_addr = "\x8e\x94\x1f\x8c\x26\x44" 
```

Comment:
Do you need to change the entire src mac? AFAIK, having 1 in the least significant bit of the first octet is what treats it as a multicast address.